packages/@react-native-windows/tester/src/js/utils/RNTesterList.windows.js

```diff
@@ -30,6 +30,11 @@ const ComponentExamples: Array<RNTesterExample> = [
     category: 'UI',
     module: require('../examples-win/DatePicker/DatePickerExample'),
   },
+  {
+    key: 'Display:NoneExample',
+    category: 'UI',
+    module: require('../examples-win/DisplayNone/DisplayNoneExample'),
```

Comment:
I think you might have missed adding the file?

Reply:
yep, struggling with .gitignore today.. should be included now. About to add the snapshot testing back in to target the new page.